### DISCUSSION TITLE
Naming Conventions is not true anymore?

### DISCUSSION THREAD

**Discussion by JCKodel** [JasperFx/marten#2667] [General]
The first article seems to be wrong. I've installed PostgreSQL 15 and created a table using PascalCase for table name and camelCase for columns, and all seems to be working just fine (contrary to what https://martendb.io/v3/documentation/postgres/naming says):

```sql
-- Table: public.Users

-- DROP TABLE IF EXISTS public."Users";

CREATE TABLE IF NOT EXISTS public."Users"
(
    id uuid NOT NULL,
    "displayName" text COLLATE pg_catalog."default" NOT NULL,
    "avatarURI" text COLLATE pg_catalog."default" NOT NULL,
    email text COLLATE pg_catalog."default" NOT NULL,
    "authProvider" text COLLATE pg_catalog."default" NOT NULL,
    "createdAt" date NOT NULL,
    "updatedAt" date NOT NULL,
    CONSTRAINT "Users_pkey" PRIMARY KEY (id)
)

TABLESPACE pg_default;

ALTER TABLE IF EXISTS public."Users"
    OWNER to postgres;
```

```sql
select *
from INFORMATION_SCHEMA.COLUMNS
where table_name = 'Users'
order by ordinal_position
```

Result:

![image](https://github.com/JasperFx/marten/assets/379339/9b86afba-82c5-4a2e-8ba9-82fb50ecf5d1)

**Comment by mysticmind** [comment]
The docs what you are referring to pertaining to topic "Postgres for SQL Server users". This portion of the docs in incomplete and has not been updated for a very quite a long time now. v3 through latest has the same content. Besides you are referring to v3 docs which possibly is written with reference to 9.5

Most likely we will remove this portion from the latest docs, re-add it if we could flesh out the content more and bring it up to date.

**Comment by mysticmind** [comment]
I am converting this to a discussion for now.
